config/coordinator/log4j2-dev.xml

```diff
@@ -28,6 +28,7 @@
           <KnownError logLevel="DEBUG" message="^.*?\bConnection reset\b.*?$" stackTrace="true" />
           <KnownError logLevel="INFO"  message="^.*?\beth_call\b.*?\bContract Call has been reverted by the EVM with the reason\b.*?$"/>
           <KnownError logLevel="INFO"  message="^.*?\brequest beyond head block\b.*?$"/>
+          <KnownError logLevel="INFO"  message="^.*?\beth_call for blob submission failed\b.*?\bmax fee per (blob )?gas less than block (blob gas|base) fee\b.*?$"/>
```

Comment:
This log level shall be addressed in the code IMO. Not, by configuration.

The configuration shall be used as a "quick fix/workaround" in prod until the log is is fixed in the code.

Reply:
This is to let the log rewriter to set the log level to INFO instead of ERROR for the reformatted error message given that the one in line 29 won't be working for insufficent gas fee error after the error message get reformatted. 

My initial thought is to align the use of log rewriter to re-set the log level, but I also think, for simplicity, we could just call `log.info()` in `LoggingHelper` if eth_call blob submission insufficent gas fee error is detected. If no objection, I'd change that to be handled in `LoggingHelper`